### DISCUSSION TITLE
Enable text compression

### DISCUSSION THREAD

**Discussion by Garlink** [supabase/supabase#3136] [Questions] [answered]
First of all, I am using Supabase for the first time and so far it's just awesome!

Although when running [Google Audit](https://web.dev/measure/) on the website that I created using Supabase and Next.js, the Lighthouse reports "Enable text compression" which affect a lot on the overall performance score. More info @ [https://web.dev/uses-text-compression/](https://web.dev/uses-text-compression/).

I've search the docs and Github issues to seek more info, but no luck so far.

Can someone help me on this?

**Comment by inian** [comment, marked as answer]
Hey @Garlink, can you provide a link to your pagespeed report or the Supabase link which it is flagging out?

**Reply by Garlink** [reply]
Sure. 

Here is the [Lighthouse report](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fwww.brunodomingos.com).

**Reply by inian** [reply]
Ah thanks for this Garlink, we will enable Brotli or GZip compression soon on the API responses

**Reply by Garlink** [reply]
Looking forward to it!

Thanks @inian and congrats to all the Supabase team for this awesome tool.

**Reply by Garlink** [reply]
Btw, @inian any idea on how soon that Brotli or Gzip will be enabled?
